app/network/Shared/Resources/Localizable.xcstrings

```diff
@@ -18816,15 +18816,15 @@
             "state" : "translated",
             "value" : "提供模式"
           }
+        },
+        "cs" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Režim poskytování"
+          }
         }
       }
     },
-    "Providing" : {
```

Comment:
Maybe this is unintentional, but it looks like you're dropping "Providing" and the Arabic translation. Could cause formatting issues.